### PR TITLE
Added redis acceptance test

### PR DESCRIPTION
**Pull request by RoyUP9**

### CODE REVIEW

File: acceptanceTests/cypress/integration/tests/Redis.js [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+                           }
+                       }
+                   });
+                });
+            });
+        });
+    });
+}
+
+function oneTimeCheck(generalDict, protocolDict, methodDict, summaryDict, entry) {
```
Comment (RoyUP9) [inline acceptanceTests/cypress/integration/tests/Redis.js:95]: rename

File: acceptanceTests/cypress/integration/tests/Redis.js [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+                           const entryNum = getEntryNumById(entry.id);
+
+                           leftTextCheck(entryNum, methodDict.pathLeft, methodDict.expectedText);
+                           leftTextCheck(entryNum, protocolDict.pathLeft, protocolDict.expectedTextLeft);
+                           if (shouldCheckSummary)
+                               leftTextCheck(entryNum, summaryDict.pathLeft, summaryDict.expectedText);
+
+                           if (!doneCheckOnFirst) {
+                                oneTimeCheck(funcDict, protocolDict, methodDict, summaryDict, entry);
+                               doneCheckOnFirst = true;
```
Comment (RoyUP9) [inline acceptanceTests/cypress/integration/tests/Redis.js:85]: indent

File: acceptanceTests/cypress/integration/tests/UiTest.js [hunk 3]
```diff
@@ -1,5 +1,11 @@
 import {findLineAndCheck, getExpectedDetailsDict} from "../testHelpers/StatusBarHelper";
-import {resizeToHugeMizu, resizeToNormalMizu, verifyMinimumEntries} from "../testHelpers/TrafficHelper";
+import {
+    leftTextCheck,
+    resizeToHugeMizu,
+    resizeToNormalMizu,
+    rightOnHoverCheck,
+    rightTextCheck
+} from "../testHelpers/TrafficHelper";
 const greenFilterColor = 'rgb(210, 250, 210)';
```
Comment (RoyUP9) [inline acceptanceTests/cypress/integration/tests/UiTest.js:9]: can be deleted?

File: acceptanceTests/cypress/integration/tests/UiTest.js [hunk 4]
```diff
@@ -1,5 +1,11 @@
...
 const redFilterColor = 'rgb(250, 214, 220)';
```
Comment (RoyUP9) [inline acceptanceTests/cypress/integration/tests/UiTest.js:10]: can be deleted?

File: acceptanceTests/cypress/integration/tests/UiTest.js [hunk 5]
```diff
@@ -1,5 +1,11 @@
 import {findLineAndCheck, getExpectedDetailsDict} from "../testHelpers/StatusBarHelper";
-import {resizeToHugeMizu, resizeToNormalMizu, verifyMinimumEntries} from "../testHelpers/TrafficHelper";
```
Comment (RoyUP9) [inline acceptanceTests/cypress/integration/tests/UiTest.js:2]: don't you need verifyMinimumEntries?

File: acceptanceTests/cypress/integration/tests/Redis.js [hunk 6]
```diff
@@ -0,0 +1,155 @@
+import {
+    leftOnHoverCheck,
+    leftTextCheck,
+    rightOnHoverCheck,
+    rightTextCheck,
+} from "../testHelpers/TrafficHelper";
+
+const possibleJsonPlaces = {
```
Comment (RoyUP9) [inline acceptanceTests/cypress/integration/tests/Redis.js:8]: maybe rename to valueTabs?

File: acceptanceTests/cypress/integration/tests/Redis.js [hunk 7]
```diff
@@ -0,0 +1,155 @@
...
+    response: 'RESPONSE',
+    request: 'REQUEST',
+    nowhere: null
```
Comment (RoyUP9) [inline acceptanceTests/cypress/integration/tests/Redis.js:11]: maybe rename nowhere to none?

**Review by gadotroee — APPROVED** [review]

**Pull request merged**